README.md

```diff
@@ -44,9 +44,9 @@ Please check the [JAM Prize website](https://jam.web3.foundation/) for participa
 
 ## Plagiarism
 
-We take licensing and the right of all teams in and outside the ecosystem to be recognised for their work very seriously. Using others' work with no attribution or indication that this was not your own work as part of a milestone delivery **will lead to immediate disqualification**. Please [reach out to us](mailto:grants@web3.foundation) before submitting if you have any doubts on how to comply with a specific license and we'll be happy to help.
+We take licensing and the right of all teams in and outside the ecosystem to be recognized for their work very seriously. Using others' work without attribution or indication that this was not your own work as part of a milestone delivery **will lead to immediate disqualification**. Please [reach out to us](mailto:jam@web3.foundation) before submitting if you have any doubts about how to comply with a specific license, and we'll be happy to help.
 
-If you suspect that a JAM Prize submission is based on plagiarized code, please also [reach out to us](mailto:grants@web3.foundation). 
+If you suspect that a JAM Prize submission is based on plagiarized code, please also reach out to jam[at]web3.foundation. 
```

Comment:
```suggestion
If you suspect that a JAM Prize submission is based on plagiarized code, please also reach out to jam@web3.foundation. 
```

Reply:
@PieWol putting the @ into brackets reduces the amount of spam that we receive, since this way at least not every bot/script out there understands that it's an email. But happy to be convinced otherwise.